DatabaseStructures

```diff
@@ -0,0 +1,26 @@
+PHONE
+id
+manufacturer
+model_name
+price
+launch_date
+screen_size
+protection_rating
+processor
+storage
+ram
+main_camera
```

Comment:
What if the phone has multiple cameras? Maybe it would be good idea to extract this into a separate table?

Reply:
No I don't think so.

Reply:
Added camera :D